platform/Stream.cpp

```diff
@@ -112,14 +103,14 @@ void Stream::rewind() {
 }
 
 int Stream::isatty() {
-    return 0;
+    return 1;
```

Comment:
Aaaaaah, I'm not sure this is really a worthwhile change to make at this point since this has the potential to break things. We're planning on deprecating this class anyways right?